src/tool/hpcrun/module-ignore-map.c

```diff
@@ -102,7 +102,7 @@
 #define PRINT(...)
 #endif
 
-#define NUM_FNS 3
+#define NUM_FNS 6
```

Comment:
add a FIXME note that indicates we will later refactor this code to make the NUM_FNS a variable and the table of functions to be looked up can be a linked list, where any GPU can indicate that its functions should be added to the module ignore map when that type of GPU is being monitored.